### ISSUE TITLE
none upgrade: kubeadm exit status 1 (bind: address already in use)

### ISSUE THREAD

**Issue by tstromberg** [kubernetes/minikube#5332]
Probable test flake https://storage.googleapis.com/minikube-builds/logs/5324/none_Linux.txt

```
                W0913 03:11:55.412882   24049 exit.go:99] Error restarting cluster: addon phase: running command: sudo env PATH=/var/lib/minikube/binaries/v1.10.13:$PATH kubeadm alpha phase addon all --config /var/tmp/minikube/kubeadm.yaml: exit status 1
                * 
                X Error restarting cluster: addon phase: running command: sudo env PATH=/var/lib/minikube/binaries/v1.10.13:$PATH kubeadm alpha phase addon all --config /var/tmp/minikube/kubeadm.yaml: exit status 1
                * 
                * Sorry that minikube crashed. If this was unexpected, we would love to hear from you:
                  - https://github.com/kubernetes/minikube/issues/new/choose
                
            start_stop_delete_test.go:123: [out/minikube-linux-amd64 start -p minikube --alsologtostderr -v=8 --cache-images=false --kubernetes-version=v1.10.13 --kvm-network=default --kvm-qemu-uri=qemu:///system --disable-driver-mounts --keep-context=false --container-runtime=docker --vm-driver=none ] failed: exit status 70
            panic.go:406: TestStartStop/group/docker failed, collecting logs ...
```

Based on the output, I feel like we may not be cleaning up a previously setup minikube test:

```
                * failed to create listener: failed to listen on 0.0.0.0:8443: listen tcp 0.0.0.0:8443: bind: address already in use
```

**Comment by tstromberg** [comment]
v1.4.0 updates this to mention the following:

```
        * Relaunching Kubernetes using kubeadm ... 
        * Problems detected in kube-controller-manager [69da8169e08c]:
          - failed to create listener: failed to listen on 0.0.0.0:10252: listen tcp 0.0.0.0:10252: bind: address already in use
        * Problems detected in kube-scheduler [96f00f4575ce]:
          - failed to create listener: failed to listen on 0.0.0.0:10251: listen tcp 0.0.0.0:10251: bind: address already in use   
```

**Comment by medyagh** [comment]
there is a small possiblity that this has been fixed by v1.6.1 if not seen again, we could close it.
